sql/core/src/test/resources/sql-tests/results/literals.sql.out

```diff
@@ -197,9 +197,9 @@ select .e3
 -- !query 20
 select 1E309, -1E309
 -- !query 20 schema
-struct<Infinity:double,-Infinity:double>
+struct<1E+309:decimal(1,-309),-1E+309:decimal(1,-309)>
```

Comment:
Yeah they can be. If you a the scientific notation they can be: http://stackoverflow.com/questions/21590590/setting-scale-to-a-negative-number-with-bigdecimal